server/src/main/scala/org/http4s/server/staticcontent/ResourceService.scala

```diff
@@ -0,0 +1,33 @@
+package org.http4s.server.staticcontent
+
+import java.io.File
+import java.util.concurrent.ExecutorService
+
+import org.http4s.server._
+import org.http4s.StaticFile
+
+import scalaz.concurrent.{Strategy, Task}
+import scalaz.OptionT
+
+
+object ResourceService {
+
+  case class Config(basePath: String,
+                    pathPrefix: String = "",
+                    bufferSize: Int = 256*1024,
+                    executor: ExecutorService = Strategy.DefaultExecutorService,
+                    cacheStartegy: CacheStrategy = NoopCacheStrategy)
+
+  /** Make a new [[org.http4s.server.HttpService]] that serves static files. */
+  private[staticcontent] def apply(config: Config): HttpService = Service.lift { req =>
+    val uri = req.uri
+    if (!uri.path.startsWith(config.pathPrefix)) Task.now(None)
+    else OptionT(
+          StaticFile.fromResource(sanitize(config.basePath + File.separator + getSubPath(uri, config.pathPrefix)))
```

Comment:
I think you're right. Just happens that most everything but windows uses ```/```. I'll change it. 
Thanks for the review.